### ISSUE TITLE
Task picks up non-scala files

### ISSUE THREAD

**Issue by smungee** [hierynomus/scalariform-gradle-plugin#3]
The `formatScala` task is picking up non-scala files as well, such as `src/main/resources/application.conf`, and then fails. Any ideas?

```
06:27:45.813 [DEBUG] [org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter] Executing actions for task ':executor-service:formatScala'.
06:27:45.813 [INFO] [org.gradle.api.Task] Reformatting the 'main' source set
06:27:45.823 [DEBUG] [org.gradle.api.Task] Formatting '[..]/src/main/resources/application.conf'
```

**Comment by noise64** [comment]
I had the same issue, created a PR for it: https://github.com/hierynomus/scalariform-gradle-plugin/pull/4

**Comment by hierynomus** [comment]
Fixed with merging #4
